libraries/websphere_jms_provider.rb

```diff
@@ -30,7 +30,7 @@ class WebsphereJmsProvider < WebsphereBase
     property :url, [String, nil], default: nil
     property :classpath_jars, [Array, nil], default: nil # full path to each jar
     property :description, [String, nil], default: nil
-    property :attributes, [Hash, nil], default: {} # Generic hash holding attributes for the provider - see : https://www.ibm.com/support/knowledgecenter/en/SS7K4U_7.0.0/com.ibm.websphere.zseries.doc/info/zseries/ae/rxml_7adminjms.html (ignore that it's for zOS)
+    property :attributes, Hash, default: {} # Generic hash holding attributes for the provider - see : https://www.ibm.com/support/knowledgecenter/en/SS7K4U_7.0.0/com.ibm.websphere.zseries.doc/info/zseries/ae/rxml_7adminjms.html (ignore that it's for zOS)
```

Comment:
```
An attempt was made to change attributes from {} to nil by calling attributes(nil). In Chef 12, this does a get rather than a set. In Chef 13, this will change to set the value to nil. at 1 location:
- /tmp/kitchen/cache/cookbooks/gol/recipes/wcsprimary.rb:997:in `block (2 levels) in from_file'
```
Set the attributes type as just `Hash` (and remove `nil`)

Reply:
Still testing...

Reply:
ignore this.